### ISSUE TITLE
Ability to create standalone notes

### ISSUE THREAD

**Issue by qriousguy** [burtonator/polar-bookshelf#667]
Hi, 
Currently, in polar, comments and flashcards are attached to a pdf or a webpage. However, both cannot be edited.
It should be possible to create standalone notes which can have comments and flashcards attached to them.
Ex: If I am sitting in class, I should be able to add a new note where I can type everything I learnt in class. Later, I can comment on it and make flashcards from it. Also, I can edit the note if I learn something new.

**Comment by burtonator** [comment]
yes.. going to add editing here soon. I agree!!! should be out in next weeks build I think.  Trying to focus more on usability now that more users are coming on...

**Comment by qriousguy** [comment]
So, in next week's build there will be a button like Add -> new note in
addition to import from disk and web page?

**Comment by burtonator** [comment]
probably not super soon but I'd like to get it in once I can take some time to clean up the annotations view.  I want to add more filtering options here so it's more usable for handling them all.
